uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/adyen/AdyenPaymentPresenter.kt

```diff
@@ -103,10 +103,9 @@ class AdyenPaymentPresenter(view: BookingPaymentMVP.View,
             when (payload.optString(AdyenPaymentView.RESULT_CODE, "")) {
                 AdyenPaymentView.AUTHORISED -> {
                     val transactionId = payload.optString(AdyenPaymentView.MERCHANT_REFERENCE, "")
-                    updateCardDetails(nonce = transactionId,
-                                      paymentData = payload.optString
-                                      (AdyenPaymentView.ADDITIONAL_DATA, null))
-                    passBackNonce(transactionId)
+                    this.sdkToken = transactionId
```

Comment:
Wondering if we could cover this logic in the unit tests

Reply:
Yes. Lots of unit tests coming up in the next PR